# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/xul/XULFrameElement.h
# Commit: 95b350572bbf
# Full Hash: 95b350572bbf079fdf6613b5a721c6ee5023fc6e
# Author: Kashav Madan <kmadan@mozilla.com>
# Date: 2020-06-23 16:15:06
# Regressor Bug: 1646563
# File Overlap Count: 1
# Description:
#   Bug 1646563 - Remove BrowserId from nsFrameLoaderOwner, r=nika
#   
#   We can just use BrowsingContext::BrowserId directly, so it's unnecessary to have
#   the field on nsFrameLoaderOwner as well.
#   
# ==============================================================================

diff -r 4eab38aba3fe -r 95b350572bbf dom/xul/XULFrameElement.h
--- a/dom/xul/XULFrameElement.h	Mon Jun 22 21:30:31 2020 +0000
+++ b/dom/xul/XULFrameElement.h	Mon Jun 22 21:46:03 2020 +0000
@@ -39,7 +39,7 @@
   already_AddRefed<nsIWebNavigation> GetWebNavigation();
   Nullable<WindowProxyHolder> GetContentWindow();
   Document* GetContentDocument();
-  uint64_t BrowserId() { return GetBrowserId(); }
+  uint64_t BrowserId();
 
   void SwapFrameLoaders(mozilla::dom::HTMLIFrameElement& aOtherLoaderOwner,
                         mozilla::ErrorResult& rv);
